.travis.yml

```diff
@@ -6,6 +6,8 @@ before_install:
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
 script:
-  - dart --checked test/run_tests.dart -p vm -p content-shell --verbose
-  - ./tool/analyze.sh
-  - ./tool/dartfmt.sh -e
\ No newline at end of file
+  - pub run dart_dev format --check
```

Comment:
yes, I believe travis runs all of the scripts in this section even if one or more of them fail